vm/src/obj/objfloat.rs

```diff
@@ -123,7 +123,8 @@ fn inner_floordiv(v1: f64, v2: f64, vm: &VirtualMachine) -> PyResult<f64> {
 
 fn inner_divmod(v1: f64, v2: f64, vm: &VirtualMachine) -> PyResult<(f64, f64)> {
     if v2 != 0.0 {
-        Ok(((v1 / v2).floor(), v1 % v2))
+        let d = (v1 / v2).floor();
+        Ok((d, v1 - (v2 * d)))
```

Comment:
This is a bit different from CPython way. This implementation is ok, but look into it if you want to do. The corresponding CPython function is `float_divmod` in `floatobject.c` file. Please tell me if you want to fix it more or want to make this merged as it is.

Reply:
Ok, I corrected it by referring to cpython.